aten/src/ATen/core/op_registration/hacky_wrapper_for_legacy_signatures.h

```diff
@@ -2,6 +2,7 @@
 
 #include <c10/util/Metaprogramming.h>
 #include <c10/util/TypeList.h>
+#include <c10/util/IntegerSequence.h>
```

Comment:
ngl: I think I would have preferred it if this was implemented via codegen rather than template